tests/unit/plugins/modules/test_user.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2020- IBM, Inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import unittest
+from unittest import mock
+import re
+import copy
+
+from ansible_collections.ibm.power_aix.plugins.modules import user
+
+from .common.utils import (
+    AnsibleFailJson, fail_json, rootdir
+)
+
+
+class TestCreateUser(unittest.TestCase):
+    def setUp(self):
+        self.module = mock.Mock()
+        self.module.fail_json = fail_json
+        self.module.run_command.return_value = (0, None, None)
+        params = dict()
+        params["name"] = "test123"
+        params["attributes"] = None
+        params["remove_homedir"] = True
+        params["change_passwd_on_login"] = True
+        params["password"] = "pass1234"
+        self.module.params = params
+        rc, stdout, stderr = 0, "sample stdout", "sample stderr"
+        self.module.run_command.return_value = (rc, stdout, stderr)
```

Comment:
Why the self.module.run_command.return_value is set twice, one in line 24 and second in line 33?